packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
+import { Logger } from 'mozlog';
+const config: ConfigType = require('../config').getProperties();
+
+export type ZipLocation = {
+  city: string;
+  country: string;
+  countryCode: string;
+  state: string;
+  stateCode: string;
+};
+
+/**
+ *
+ * @param zip
+ * @param country - Two-letter country code ([ISO 3166-1 alpha-2] or corresponding Country name
+ * @returns
+ */
+export async function getLocationFromZip(
+  log: Logger,
+  zip: string,
+  country: string = 'US'
+): Promise<ZipLocation | {}> {
+  const client = new Client();
+  const countryName =
+    country.length === 2 ? countries.getName(country, 'en') : country;
+
+  try {
+    const {
+      data: { results, status },
+    } = await client.geocode({
+      params: {
+        address: `${zip},${countryName}`,
+        key: config.subscriptions.googleMapsApiKey,
```

Comment:
Can we create the `new Client` passing in the API key instead of passing it in with every call?